### PR TITLE
Add CFBundleURLSchemes when building for iOS in debug.

### PR DESCRIPTION
**Pull request by TomaNikolov**
We will use custom URL scheme to open iOS app from the browser. 
This way we will support app restart on Windows. 
Related to:  https://github.com/NativeScript/ios-runtime/issues/746

### CODE REVIEW

File: lib/services/ios-project-service.ts [hunk 1]
```diff
@@ -704,7 +704,7 @@ We will now place an empty obsolete compatability white screen LauncScreen.xib f
 		return Promise.resolve();
 	}
 
-	private async mergeInfoPlists(projectData: IProjectData): Promise<void> {
+	private async mergeInfoPlists(release: boolean, projectData: IProjectData): Promise<void> {
```
Comment (rosen-vladimirov) [inline lib/services/ios-project-service.ts]: According to our coding guidelines, we prefer passing objects with the required boolean properties, which will make the calling of the function more readable. In this case I suggest you to use the `IRelease` interface, for example:
```TypeScript
private async mergeInfoPlists(buildOption: IRelease, projectData: IProjectData): Promise<void> {
```
In the method body you'll check if `buildOptions.release` is true. The caller will pass `{ release: <value> `

Comment (rosen-vladimirov) [inline lib/services/ios-project-service.ts]: Also I prefer having the projectData as a first argument... but I'm not sure why I prefer it

File: lib/services/ios-project-service.ts [hunk 2]
```diff
@@ -672,7 +672,7 @@ We will now place an empty obsolete compatability white screen LauncScreen.xib f
 	}
 
 	public async processConfigurationFilesFromAppResources(release: boolean, projectData: IProjectData): Promise<void> {
-		await this.mergeInfoPlists(projectData);
+		await this.mergeInfoPlists({release}, projectData);
```
Comment (Mitko-Kerezov) [inline lib/services/ios-project-service.ts]: Could you format this (either put spaces around the curly brackets or use VS Code's formatting)

File: lib/services/ios-project-service.ts [hunk 3]
```diff
@@ -704,7 +704,7 @@ We will now place an empty obsolete compatability white screen LauncScreen.xib f
 		return Promise.resolve();
 	}
 
-	private async mergeInfoPlists(projectData: IProjectData): Promise<void> {
+	private async mergeInfoPlists(buildOption: IRelease, projectData: IProjectData): Promise<void> {
```
Comment (Mitko-Kerezov) [inline lib/services/ios-project-service.ts]: I'd rather we name `buildOption` argument to `buildOptions` (plural) as we could put more options in it in the future

**Pull request merged**